# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/NotificationParent.h
# Commit: f318fe78ff14
# Full Hash: f318fe78ff1444fe0c3c46495f749720c9c53c16
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-08 04:19:40
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 7: Settle ShowResolver r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227692
# ==============================================================================

diff -r 3daeacd08336 -r f318fe78ff14 dom/notification/NotificationParent.h
--- a/dom/notification/NotificationParent.h	Thu Nov 07 17:32:05 2024 +0000
+++ b/dom/notification/NotificationParent.h	Thu Nov 07 17:32:06 2024 +0000
@@ -55,6 +55,8 @@
   }
   void MaybeInitAlertName();
 
+  Maybe<NotificationParent::ShowResolver> mResolver;
+
   NotNull<nsCOMPtr<nsIPrincipal>> mPrincipal;
   NotNull<nsCOMPtr<nsIPrincipal>> mEffectiveStoragePrincipal;
   bool mIsSecureContext;
